### ISSUE TITLE
Server Randomly Crashing (very frequently)

### ISSUE THREAD

**Issue by Mangonic** [PaperMC/Paper#6556]
### Stack trace

https://paste.gg/p/anonymous/d086de01c6d743f398e20b90a0d797ab

### Plugin list

Plugins (45): AnvilEnhanced, AuctionHouse, AureliumSkills, BetterRTP, BungeeGuard, ChatControl, Citizens, ClearLag, CombatLogX, CommandAliases, CommandPanels, CoreProtect, CrashAPI, CustomEnderChest, DeathMessages, DeluxeMenus, DiscordSRV, Essentials, EssentialsChat, EssentialsProtect, EssentialsSpawn, Gatekeeper*, GoldenCrates, HeadDatabase, HolographicDisplays, IllegalStack, Landlord, LuckPerms, Multiverse-Core, Multiverse-Portals, Negativity, NexEngine, NoCheatPlus, PlaceholderAPI, PlayerParticles, PlayerProfiles, RealScoreboard, ShopGUIPlus, ShopGUIPlus-SellGUI, SirBlobmanCore (SirBlobmanAPI, XSeries), TAB, Vault, ViaVersion, WorldEdit, WorldGuard

### Paper version

[Fri, 3. Sep 2021 22:32:50 CEST INFO] Checking version, please wait...
[Fri, 3. Sep 2021 22:32:50 CEST INFO] This server is running Paper version git-Paper-788 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
You are running the latest version
Previous version: git-Paper-783 (MC: 1.16.5)

### Agreements

- [X] I am running the latest version of Paper available from https://papermc.io/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] My version of Minecraft is supported by Paper.

### Other

So basically, like 2 days ago my server decided to crash like every 20min-2h out of nowhere. The players ingame will get the message "[Proxy] Proxy Restarting" even if it doesn't, only my main server is crashing.

In the Error Log, the "problematic frame" always is something else every crash. I don't know if that says something, but in my eyes it's weird. I really don't know what is causing my server to crash, it can't be the RAM or the CPU usage since I have allocated more than enough.

**Comment by e-im** [comment]
This issue isn't a paper bug, please come by [our discord](https://discord.gg/papermc) for support with this issue.
